Library/Discworld Roleplaying Game/Discworld Roleplaying Game Equipment.eqp

```diff
@@ -0,0 +1,4429 @@
+{
+	"type": "equipment_list",
+	"version": 4,
+	"rows": [
+		{
+			"id": "078b1c93-7707-4c60-a064-0bb985e888d0",
+			"type": "equipment",
+			"description": "Agatean Numknuts",
+			"tech_level": "3",
+			"tags": [
+				"Melee Weapons"
+			],
+			"quantity": 1,
+			"value": 1,
+			"weight": "2 lb",
+			"weapons": [
+				{
+					"id": "21c51b80-1a28-4d29-abda-66fab8375f66",
+					"type": "melee_weapon",
+					"damage": {
+						"type": "cr",
+						"st": "sw",
+						"base": "1"
+					},
+					"strength": "7",
+					"usage": "Swing",
+					"reach": "1",
+					"parry": "0U",
+					"defaults": [
+						{
+							"type": "skill",
+							"name": "FLAIL"
```

Comment:
You have all the weapon skill names in all caps.  This is inconsistent with other eqp files, which have them in title case, like "Flail".